addons/mail/static/src/components/composer/composer.js

```diff
@@ -207,7 +207,7 @@ class Composer extends Component {
             }
             return;
         }
-        await this.composer.postMessage();
+        await this.composer.thread.sender.insertMessageToBeSent();
```

Comment:
I would expect an "insert message" method to be given the message to insert. It's not good that it has to look it up (we want to allow having multiple composer models per thread, one per actually displayed composer).